Missing declarations

### Environment

Node v22
MacOS 15.3.1

### Reproduction

https://stackblitz.com/edit/node-tyzqzyk5?file=src%2Findex.ts

### Describe the bug

After updating to `v0.2.x`, the declarations are missing.

### Additional context

_No response_

### Logs

```sh
> node-starter@0.0.0 build
> tsc

src/index.ts:1:23 - error TS2307: Cannot find module 'srvx' or its corresponding type declarations.

1 import { serve } from 'srvx';
                        ~~~~~~


Found 1 error in src/index.ts:1
```

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "license": "MIT",
   "sideEffects": false,
   "type": "module",
+  "types": "./dist/types.d.mts",
   "exports": {
     "./types": "./dist/types.d.mts",
     "./deno": "./dist/adapters/deno.mjs",

